jax/_src/api.py

```diff
@@ -85,6 +85,8 @@
 
 traceback_util.register_exclusion(__file__)
 
+_dtype = partial(dtypes.dtype, canonicalize=True)
```

Comment:
Is it really worth having this alias. It seems that there is a single use in this file and I assume no uses in other files.

Reply:
I count 13 uses in this file, 12 of them existing before this PR... I could do that refactor if you think it would be better?

Reply:
(Followup: this comment seems similar to below - I'm not sure how much there is to be gained from removing these helpers)